addons/addon-base-raas/packages/base-raas-services/lib/alb/alb-service.js

```diff
@@ -0,0 +1,298 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const _ = require('lodash');
+const Service = require('@aws-ee/base-services-container/lib/service');
+
+const settingKeys = {
+    domainName: 'domainName',
+};
+
+class ALBService extends Service {
```

Comment:
Missing unit tests.

Reply:
I'd recommend adding the unit tests in this PR itself, since the automated code coverage check would fail.